zebra-consensus/src/checkpoint.rs

```diff
@@ -492,15 +492,7 @@ where
         );
 
         let is_checkpoint = self.checkpoint_list.contains(height);
-        tracing::trace!(?height, ?hash, ?is_checkpoint, "Queued block");
-
-        // TODO(teor):
-        //   - Remove this log once the CheckpointVerifier is working?
-        //   - Modify the default filter or add another log, so users see
-        //     regular download progress info (vs verification info)
-        if is_checkpoint {
-            tracing::info!(?height, ?hash, ?is_checkpoint, "Queued checkpoint block");
```

Comment:
I find this log really useful for diagnostics, can we keep it?

Reply:
My preference would be to remove it, since the checkpoint verifier is working now, and download progress is better monitored using the metrics integration.

Reply:
The metrics integration works for some users, but others won't necessarily have it set up.

If we don't log anything, we'll get bug reports which say that Zebra isn't making any progress.

Reply:
Ah, right, but we're still getting a log for each verified checkpoint, so this isn't a big issue.